src/components/DetailsPage.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+const DetailsPage = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  return (
+    <div className="details-page">
+      <header className="home-header">
+        <div className="logo">
+          <button type="button" className="btn-logo" onClick={() => navigate(-1)}>
+            {' '}
+            <ArrowBackIcon />
+            Go back
+          </button>
+        </div>
+      </header>
+      <div className="details-page-details">
+        <div>
+          <img src={location.state.flag} alt="Country flag" />
+        </div>
+        <div className="state-details">
+          <h2>{location.state.name}</h2>
+          <p>
+            Native Name:
+            {location.state.nativeName}
+          </p>
+          <p>
+            Population:
+            {location.state.population}
+          </p>
+          <p>
+            Continent:
+            {location.state.region}
+          </p>
```

Comment:
- [ ] You should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

* Typography: font face, size, and weight
* Layout: composition and space between elements.Student should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

Typography: font face, size, and weight
Layout: composition and space between elements.

![image](https://user-images.githubusercontent.com/79058364/211159817-169727df-5799-4080-8d09-5b42a1c35261.png)

- [ ] Actually, your `DetailsPage` is not styling yet, Kindly make sure the `DetailsPage`  style match the one provided 🙃
- [ ] Kindly improve your Homepage styling as well to match the original one 👍🏻

Reply:
- [ ] [REQUIRED] Kindly note that the width of the flags is not consistent. Let's fix this in order to have a better UI. Take a look at the template provided.  I would advise you to give the flag a uniform padding or margin to the left and right so that the flags can be positioned uniformly as in the template.

### Your design
<img width="339" alt="Screenshot 2023-01-07 at 19 31 21" src="https://user-images.githubusercontent.com/58692590/211165683-fecfd166-266b-4962-bfe9-3f7620717047.png">